input/pagecontent/index.md

```diff
@@ -50,13 +50,34 @@ The RESTful interaction requirements for AU eRequesting actors, such as read and
 
 The diagram below illustrates the interactions between the various actors:
 
+<figure>
 <div> 
   <img src="au-erequesting-actor-interactions.svg" alt="FHIR RESTful read interactions" style="width:80%"/>
 </div>
+<figcaption>Figure 1: FHIR RESTful interactions</figcaption>
+</figure>
 <br/>
-*Figure 1: FHIR RESTful interactions*
+
+The AU eRequesting Placer actor can create and update ServiceRequest details by calling the AU eRequesting Server actor.
+
+The AU eRequesting Patient Access actor can search for ServiceRequest details by calling the AU eRequesting Server actor.
+
+The AU eRequesting Filler actor can retrieve ServiceRequest details by calling the AU eRequesting Server actor.
+
+Actors operate in a typical sequence where placers initiate a request by creating (and updating) service requests, patients may access details of their requests and fillers retrieve service request details to enable fulfillment of the requests. 
+
+This sequence is shown in the diagram below:
+
+<figure>
+<div> 
+    <img src="actors-sequence.png" alt="Actors Typical Sequence" style="width:611px;margin-left:auto;margin-right:auto;display:block"/>
+</div>
+<figcaption>Figure 2: Actors Typical Sequence</figcaption>
```

Comment:
Figures aren't in fig captions.

Change to be `*Figure 2: Actors Typical Sequence*`